### ISSUE TITLE
Issue with List classnames

### ISSUE THREAD

**Issue by zabojad** [kiroukou/haxe-lodash-externs#2]
I'm having this error:
```
src/myproj/view/page/MyPage.hx:37: characters 1-7 : Invalid number of type parameters for ons.react.List
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
src/myproj/view/page/MyPage.hx:37: characters 1-7 : Invalid number of type parameters for ons.react.List
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
/usr/local/lib/haxe/lib/lodash/git/./lib/lodash/Lodash.hx:139: characters 14-31 : Unknown identifier : __cleanLodashList
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
/usr/local/lib/haxe/lib/lodash/git/./lib/lodash/Lodash.hx:140: characters 10-27 : Unknown identifier : __cleanLodashList
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
/usr/local/lib/haxe/lib/lodash/git/./lib/lodash/Lodash.hx:57: characters 10-23 : Unknown identifier : __cleanLodashList
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
/usr/local/lib/haxe/lib/lodash/git/./lib/lodash/Lodash.hx:58: characters 15-28 : Unknown identifier : __cleanLodashList
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
/usr/local/lib/haxe/lib/lodash/git/./lib/lodash/Lodash.hx:60: characters 6-19 : Unknown identifier : __cleanLodashList
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
/usr/local/lib/haxe/lib/lodash/git/./lib/lodash/Lodash.hx:74: characters 10-29 : Unknown identifier : __cleanLodashStaticList
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
/usr/local/lib/haxe/lib/lodash/git/./lib/lodash/Lodash.hx:75: characters 15-34 : Unknown identifier : __cleanLodashStaticList
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
/usr/local/lib/haxe/lib/lodash/git/./lib/lodash/Lodash.hx:77: characters 6-25 : Unknown identifier : __cleanLodashStaticList
src/myproj/view/page/MyPage.hx:38: lines 38-113 : Defined in this class
```

on a react class that uses a React component also named `List` (but which does not lay in the root package):
```
package blabla;

import react.ReactComponent;
import react.ReactMacro.jsx;

import ons.react.List;

// ...

@:build(lib.lodash.Lodash.build())
class MyPage extends ReactComponentOfProps<MyPageProps> {
  // ...
  overide function render(){
    return jsx('
      <List>
        { content() }
      </List>
    ');
  }
}
```

This seems to be because the lodash haxe lib uses the haxe `List` class and there is a conflict with the `ons.react.List` used here... How should we edit the lodash extern so that such conflicts do not happen?

For now, I workaround the problem in my project like this but it would be nice to not having this error in the first place:
```
// ...
    var lc = ons.react.List;
    return jsx('
      <$lc>
        { content() }
      </$lc>
    ');
```

@kiroukou

**Comment by kiroukou** [comment]
Hello @zabojad 
This issue should be fixed with commit e44352c20f624421925a8ce7fc29bee2cd9c8ca9

Let me know how it goes !

**Comment by zabojad** [comment]
it works, thank you!
